readchar/__init__.py

```diff
@@ -1,3 +1,8 @@
+# -*- coding: utf-8 -*-
+# This file is based on this gist:
+# http://code.activestate.com/recipes/134892/
+# So real authors are DannyYoo and company.
```

Comment:
Not really.
This file is already mine :)
The pieces taken from that gist are the ones under readchar_*, and they have been modified a lot, but I like to maintain the authority.
Anyways, the comment is welcome here too.